src/pybind/mgr/ssh/module.py

```diff
@@ -35,6 +36,29 @@
                       'User root\n'
                       'StrictHostKeyChecking no\n')
 
+def handle_exception(prefix, cmd_args, desc, perm, func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except (orchestrator.OrchestratorError, ImportError) as e:
+            # Do not print Traceback for expected errors.
+            return HandleCommandResult(-errno.ENOENT, stderr=str(e))
+        except NotImplementedError:
+            msg = 'This Orchestrator does not support `{}`'.format(prefix)
+            return HandleCommandResult(-errno.ENOENT, stderr=msg)
+
+    return CLICommand(prefix, cmd_args, desc, perm)(wrapper)
+
+def _cli_command(perm):
+    def inner_cli_command(prefix, cmd_args="", desc=""):
+        return lambda func: handle_exception(prefix, cmd_args, desc, perm, func)
+    return inner_cli_command
+
+_read_cli = _cli_command('r')
+_write_cli = _cli_command('rw')
```

Comment:
(feeling kinda honored for making us of it, but)

Copied from `orchestrator_cli`.  I'd be :+1: to have this available in the `orchestrator.py` interface.